cryptoinvestor/models/user.py

```diff
@@ -7,3 +8,44 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
+        self.id = id
+        self.rates = {}
```

Comment:
I don't think an instance of a user should have rates (although I'm not sure rates of what)

Reply:
I think it can be removed

Reply:
Yes remove it please